charts/consul/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: consul
-version: 1.1.0-dev
-appVersion: 1.14.4
+version: 1.1.0
+appVersion: 1.15.0
 kubeVersion: ">=1.21.0-0"
```

```suggestion
kubeVersion: ">=1.22.0-0"
```

We should bump this since most K8s distros (i.e. EKS) now support 1.22.x and above.

---

I you think so, I will bump it.

---

Yes please.